src/audio.c

```diff
@@ -1995,3 +1995,128 @@ int audio_print_rtpstat(struct re_printf *pf, const struct audio *a)
 
 	return err;
 }
+
+
+/*
+ *
+ * RTCP-XR PUBLISH
+ * Reference:  RFC 6035
+ *
+ */
+char * audio_print_rtcpxr(char * msg, int msg_len,
+       const struct audio *a, const char * cid)
```

Comment:
I think it is better to return `int` here for the error code.

perhaps we can use a `%H` compatible printer instead ?